src/bind/input/input.cpp

```diff
@@ -13,17 +13,18 @@ void init_input(py::module_ &m) {
 
   py::class_<vroom::Input>(m, "Input")
       .def(
-          py::init([](const vroom::io::Servers &servers, vroom::ROUTER router) {
-            return new vroom::Input(servers, router);
+          py::init([](const vroom::io::Servers &servers, vroom::ROUTER router, bool apply_TSPFix) {
+            return new vroom::Input(servers, router, apply_TSPFix);
           }),
           "Class initializer.",
           py::arg("servers") = std::map<std::string, vroom::io::Servers>(),
-          py::arg("router") = vroom::ROUTER::OSRM)
+          py::arg("router") = vroom::ROUTER::OSRM,
+          py::arg("apply_TSPFix") = false)
       .def_readonly("jobs", &vroom::Input::jobs)
       .def_readonly("vehicles", &vroom::Input::vehicles)
       .def("_from_json", &vroom::io::parse, py::arg("json_string"),
            py::arg("geometry"))
-      .def("_set_amount_size", &vroom::Input::set_amount_size)
+      /* .def("_set_amount_size", &vroom::Input::set_amount_size) */
```

Comment:
@jcoupey, seems like `vroom::Input::set_amount_size` is removed. Is the feature deprecated or just moved somewhere else?